pkg/sql/parser/datum.go

```diff
@@ -963,6 +963,7 @@ const (
 	timestampWithOffsetSecondsZoneFormat  = timestampWithOffsetZoneFormat + ":00"
 	timestampWithNamedZoneFormat          = timestampFormat + " MST"
 	timestampRFC3339NanoWithoutZoneFormat = "2006-01-02T15:04:05"
+	timestampSequelizeFormat              = timestampFormat + ".000 -07:00"
```

Comment:
I’m not a huge fan of this name, but I couldn’t stomach following the convention with `timestampWithFractionalSecondsAndOffsetSecondsZoneFormat`